go/client/cmd_pprof.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2018 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"errors"
+	"time"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+	"golang.org/x/net/context"
+)
+
+type CmdPprofTrace struct {
+	libkb.Contextified
+	traceFile     string
+	traceDuration time.Duration
+}
+
+func (c *CmdPprofTrace) ParseArgv(ctx *cli.Context) error {
+	args := ctx.Args()
+	if len(args) != 1 {
+		return errors.New("Trace needs a single file path")
+	}
+	c.traceFile = args.First()
+	c.traceDuration = ctx.Duration("duration")
+	if c.traceDuration <= 0 {
+		c.traceDuration = 5.0
```

Comment:
error instead?

Reply:
Done. It was a bug anyway, since 5.0 is in nanoseconds, not seconds!